### ISSUE TITLE
feat: Anaphora checking

### ISSUE THREAD

**Issue by elijah-potter** [Automattic/harper#19]
Not sure what we can do here. There are some situations where repetition of the same word at the beginning of a sentence is a literary device (anaphora), and others where it comes off as weird and wrong.

How can we notify the user when they do that latter?

**Comment by grantlemons** [comment]
Code action to ignore?

**Comment by hippietrail** [comment]
Warn that it's possibly wrong with an action to ignore just this one, for the whole document, for the whole project, or everywhere.

**Comment by elijah-potter** [comment]
Yeah, I'm not sure whether this kind of rule is actually helpful. Is this a feature anyone actually wants?
